hs_core/templates/pages/genericresource.html

```diff
@@ -1357,7 +1358,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                 var t = $('#extraMetaTable').DataTable();
                 var row_to_edit = t.row("#" + row_id_str);
                 var oldname = row_to_edit.data()[0];
-                var oldvalue = row_to_edit.data()[1];
+                var oldvalue = $("<div/>").html(row_to_edit.data()[1]).text();
```

Comment:
I think it is better to have it just in case something messes up with the HTML input.